circlecheck.lic

```diff
@@ -498,6 +498,9 @@ def main
 
   $debug_mode_cc = args.debug
 
+  # Get settings so we can check for yaml prettyprint true or false
+  @settings = get_settings
```

Comment:
If `@settings` is only going to be used on line 516, then I don't think it should be a local var. I instead suggest moving line 516 up here and passing the `prettyprint` value into `display_requirements`.

Reply:
Fair, I'll change this.